src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS/Http.config

```diff
@@ -0,0 +1,1034 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this here? Use IIS.config or IISExpress.config.

Reply:
I don't want to break any downlevel dependencies by having a different config file.

Reply:
ic